sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleExponentialHistogramBuckets.java

```diff
@@ -249,6 +245,34 @@ int getScaleReduction(long newStart, long newEnd) {
     return scaleReduction;
   }
 
+  private static int getIndexByLogarithm(double value, int scale) {
+    double scaleFactor = Math.scalb(1D / Math.log(2), scale);
```

Comment:
Let's store `logScaleFactor` in a field like it was before to avoid recalculating every time

Reply:
I've gone ahead and done this.